### ISSUE TITLE
cloning repositories might need to wait and retry

### ISSUE THREAD

**Issue by maxyz** [sociomantic-tsunami/git-hub#214]
This is a forward of the issue reported in debian#859590 (https://bugs.debian.org/859590):

```
Package: git-hub
Version: 0.11.0-1
Severity: normal
Usertags: race

I tried to fork the youtube-dl repository and that worked but then
failed to clone the repository because git-hub didn't wait until the
repository was ready and public before downloading it. If it gets a
failure it should poll until the repo is ready, with the usual backoff.

$ git hub clone https://github.com/rg3/youtube-dl/
Checking for existing fork / forking...
Fork at https://github.com/pabs3/youtube-dl
Cloning https://github.com/pabs3/youtube-dl.git to youtube-dl
Error: git clone --quiet -- https://github.com/pabs3/youtube-dl.git youtube-dl failed (return code: 128)
fatal: remote error: access denied or repository not exported: /6/nw/64/fc/0e/1039520/87255687.git
$ git hub clone https://github.com/rg3/youtube-dl/
Checking for existing fork / forking...
Fork at https://github.com/pabs3/youtube-dl
Cloning https://github.com/pabs3/youtube-dl.git to youtube-dl

```

**Comment by leandro-lucarella-sociomantic** [comment]
Strange, I had the feeling I already addressed this in the past!

**Comment by leandro-lucarella-sociomantic** [comment]
Fixed, will be released in the upcoming v0.11.1.
